terraform/cloudwatch_dashboard.mirror.template.json.py

```diff
@@ -0,0 +1,581 @@
+from textwrap import (
+    dedent,
+)
+
+from azul import (
+    config,
+)
+
+dashboard_body = {
+    'widgets': [
+        {
+            'height': 6,
+            'width': 12,
+            'y': 0,
+            'x': 0,
+            'type': 'log',
+            'properties': {
+                'query': dedent(f'''\
+            SOURCE '/aws/lambda/{config.indexer_function_name('mirror')}'
+            | fields strcontains(@message, 'Worker successfully handled') as success,
+                     strcontains(@message,'Worker failed to handle message') as failure,
+                     strcontains(@message,'Task timed out after') as timeout
+            | filter failure > 0 or success > 0 or timeout > 0
+            | stats sum(success) as Successes,
+                    sum(failure + timeout) as Failures
+                    by bin(5min)
+        '''),
```

Comment:
Something's wrong with the indentation here, and throughout this file.

```suggestion
                'query': dedent(f'''\
                    SOURCE '/aws/lambda/{config.indexer_function_name('mirror')}'
                    | fields strcontains(@message, 'Worker successfully handled') as success,
                             strcontains(@message,'Worker failed to handle message') as failure,
                             strcontains(@message,'Task timed out after') as timeout
                    | filter failure > 0 or success > 0 or timeout > 0
                    | stats sum(success) as Successes,
                            sum(failure + timeout) as Failures
                            by bin(5min)
                '''),
```